### PR TITLE
Make single deletion messages for moderators obey the setting

### LINKED ISSUE #3119: 'Show Deletions of Single Messages` not handling mod only versions of the messages.
**Describe your issue**

The setting does not handle the variation of the message that mods receive, stating who deleted the message.

1. Delete a message with the setting on - see two messages related to this deletion.
2. Delete a message with the setting off - still see one message related to this deletion.

**Screenshots**

setting on
image

setting off
image

Note: `The message from x is deleted` is unrelated and specific to the fact that you pressed the button, only you can see it.

**OS and Chatterino Version**
windows 10 | Tested on ae9f92de & `#3115` checked out and built (Rate Limit JOIN's PR)

**note** 
it was an oversight btw 
image

### PR DESCRIPTION
**Pull request by Mm2PL**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Closes #3119

### CODE REVIEW

**Review by Felanbird — APPROVED** [review]
Functionality works as intended 👍 
image

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 ## Unversioned
 
+- Fix: Notifications for moderators about other moderators deleting messages can now be disabled (#3121)
```
Comment (ALazyMeme) [inline CHANGELOG.md:5]:
```suggestion
- Bugfix: Notifications for moderators about other moderators deleting messages can now be disabled. (#3121)
```

**Review by ALazyMeme — APPROVED** [review]
image

**Comment by Felanbird** [comment]
As a note, this fix moves the total messages per deletion for mods from 2 to 1.

Setting on in current master
image

Setting on in this PR
image

**Review by pajlada — APPROVED** [review]

**Pull request merged**